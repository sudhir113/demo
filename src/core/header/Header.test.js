import React from "react";
import { shallow } from "enzyme";
import Header from "./Header";

describe("Header", () => {
  it("should render the component with no props", () => {
    expect(shallow(<Header />)).toMatchSnapshot();
  });

  it("renders without crashing", () => {
    shallow(<Header />);
  });
});
