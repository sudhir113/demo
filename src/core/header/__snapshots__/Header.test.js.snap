// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render the component with no props 1`] = `
<div>
  <Link
    to="/"
  >
    Home
  </Link>
  <Link
    to="/about"
  >
    About
  </Link>
</div>
`;
